sabnzbd/cfg.py

```diff
@@ -406,6 +406,7 @@ def validate_notempty(root, value, default):
 preserve_paused_state = OptionBool("misc", "preserve_paused_state", False)
 enable_par_cleanup = OptionBool("misc", "enable_par_cleanup", True)
 process_unpacked_par2 = OptionBool("misc", "process_unpacked_par2", True)
+enable_multipar = OptionBool("misc", "enable_multipar", True)
```

Comment:
Can you try having it disabled by default? Then it can run the unit tests on ut.

Reply:
sure thing